tests/test_docker_image_workflow.py

```
@@ -0,0 +1,303 @@
+which also parses workflow files with regular expressions instead of a
+YAML library).
+"""
+
+import re
+import subprocess
+import unittest
+from pathlib import Path
+
+REPO_ROOT = Path(
+    subprocess.check_output(
+        ["git", "rev-parse", "--show-toplevel"], text=True
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)